addon/adapters/rest.js

```diff
@@ -1245,6 +1245,15 @@ function ajaxError(adapter, payload, requestData, responseData) {
   return error;
 }
 
+// Adapter abort error to include any relevent info, e.g. request/response:
+function handleAbort(requestData, responseData) {
+  let { method, url, errorThrown } = requestData;
+  let { status } = responseData;
+  let msg = `Request failed: ${method} ${url} ${errorThrown || ''}`;
+  let errors = [{ title: 'Adapter Error', detail: msg.trim(), status }];
+  return new AbortError(errors);
```

Comment:
Having this error with the url, and error response if available would be much better. 

Currently we only use `new AbortError();`